### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by stephanie1997-kek**

### CODE REVIEW

**Comment by stephanie1997-kek** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>MalayGroup3</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: using "

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```
Comment (saobv-2223) [inline app/javascript/packs/application.js]: using '

**Comment by stephanie1997-kek** [comment]
ready

**Pull request merged**